app/src/features/surveys/view/survey-spatial/components/animal/SurveySpatialAnimalCapturePopup.tsx

```diff
@@ -1,37 +1,41 @@
-import { IStaticLayerFeature } from 'components/map/components/StaticLayers';
 import { DATE_FORMAT } from 'constants/dateTimeFormats';
 import dayjs from 'dayjs';
 import { SurveyMapPopup } from 'features/surveys/view/SurveyMapPopup';
 import { useCritterbaseApi } from 'hooks/useCritterbaseApi';
 import useDataLoader from 'hooks/useDataLoader';
-import { ICaptureResponse } from 'interfaces/useCritterApi.interface';
 import { Popup } from 'react-leaflet';
 
-export interface ISurveySpatialAnimalCapturePopupProps {
-  feature: IStaticLayerFeature;
+interface ISurveySpatialAnimalCapturePopupProps {
+  captureId: string;
 }
 
 /**
- * Renders a popup for animal capture data on the map.
+ * Returns information about a critter capture record, shown when a capture point is clicked on the map
  *
  * @param {ISurveySpatialAnimalCapturePopupProps} props
- * @return {*}
+ * @returns {*}
  */
 export const SurveySpatialAnimalCapturePopup = (props: ISurveySpatialAnimalCapturePopupProps) => {
-  const { feature } = props;
+  const { captureId } = props;
 
   const critterbaseApi = useCritterbaseApi();
 
-  const captureDataLoader = useDataLoader((captureId) => critterbaseApi.capture.getCapture(captureId));
+  const captureDataLoader = useDataLoader(() => critterbaseApi.capture.getCapture(captureId));
+  const animalDataLoader = useDataLoader(critterbaseApi.critters.getCritterSimple);
+
+  const formatPopupMetadata = () => {
+    if (!captureDataLoader.data || !animalDataLoader.data) return [];
+
+    const { capture_date, capture_time, capture_location } = captureDataLoader.data;
+    const { animal_id } = animalDataLoader.data;
 
-  const getCaptureMetadata = (capture: ICaptureResponse) => {
     return [
-      { label: 'Capture ID', value: String(capture.capture_id) },
-      { label: 'Date', value: dayjs(capture.capture_date).format(DATE_FORMAT.LongDateTimeFormat) },
-      { label: 'Time', value: String(capture.capture_time ?? '') },
+      { label: 'Nickname', value: animal_id },
+      { label: 'Date', value: dayjs(capture_date).format(DATE_FORMAT.LongDateTimeFormat) },
+      { label: 'Time', value: String(capture_time ?? '') },
       {
         label: 'Coordinates',
-        value: [capture.release_location?.latitude ?? null, capture.release_location?.longitude ?? null]
+        value: [capture_location?.latitude, capture_location?.longitude]
```

Comment:
With this change, If either latitude or longitude is undefined then this will blow up, because the `.filter` is only check that they are not null.

We either need to cast these to null like it did before OR add an undefined check to the filter.

```
.filter((coord): coord is number => coord !== null && coord !== undefined)
```

and actually we have a helper function for this case

```
.filter((coord): coord is number => isDefined(coord))
```